data/compliance_profiles/CentOS/7/disa_stig.json

```diff
@@ -517,15 +550,15 @@
           "SRG-OS-000070-GPOS-00038",
           "CCI-000193"
         ],
-        "value": "-1"
+        "value": -1
       },
       "pam::cracklib_maxclassrepeat": {
         "identifiers": [
           "RHEL-07-010160",
           "SRG-OS-000072-GPOS-00040",
           "CCI-000195"
         ],
-        "value": 4
+        "value": 2
```

Comment:
From what I can tell, this number is arbitrary, but I only see 4 listed online, where did you get 2?